### PR TITLE
Add feature: Plugins update

### PR DESCRIPTION
**Pull request by NickRimmer**
Hi guys,

I'm thrilled to suggest a new feature for Insomnia! This feature will make updating installed plugins easier and more intuitive for all users. Previously, the update process required a bit of maneuvering, as users had to reinstall their plugins. I've essentially added a touch of UI enhancement to streamline this process. I'm eager to hear everyone's feedback on this!

### CODE REVIEW

File: packages/insomnia/src/ui/components/settings/plugins.tsx [hunk 1]
```diff
@@ -160,6 +232,33 @@ export const Plugins: FC = () => {
...
+            </button>
+          )}
+
+          <div>
+            <p style={{ marginBottom: 5 }}>{updates.message}</p>
+            <ul>
+              {updates.plugins.map(update => (
+                <li key={update.name}>
+                  <span style={{ marginRight: 5 }}>
+                    {update.status === 'in-progress' && (<i className="fa fa-refresh fa-spin" title="Updating in progres" />)}
```
Comment (bbbco) [inline packages/insomnia/src/ui/components/settings/plugins.tsx]: `progres` => `progress`

File: packages/insomnia/src/ui/components/settings/plugins.tsx [hunk 2]
```diff
@@ -160,6 +232,33 @@ export const Plugins: FC = () => {
...
+                    {update.status === 'error' && (<i className="fa fa-exclamation-triangle" style={{ color: 'var(--color-warning)' }} title={update.statusMessage ?? 'Plugin cannot be udpated'} />)}
```
Comment (bbbco) [inline packages/insomnia/src/ui/components/settings/plugins.tsx]: `Plugin cannot be udpated` => `Plugin cannot be updated`

File: packages/insomnia/src/ui/components/settings/plugins.tsx (hunk 1 above)
Comment (NickRimmer) [reply packages/insomnia/src/ui/components/settings/plugins.tsx]: Oh, thank you, updated (;

File: packages/insomnia/src/ui/components/settings/plugins.tsx (hunk 2 above)
Comment (NickRimmer) [reply packages/insomnia/src/ui/components/settings/plugins.tsx]: Thank you \~_\~ updated

**Comment by HeatherLemieux** [comment]
I think this is a great feature, thank you so much for your work on this! However, `Check Updates` implies that it will only `Check` whether there are updates or not and present that information to me. I would actually prefer that approach as well. I don't want all my plugins to be updated without me being able to first review. Ideally, I would be able to click `Check Updates`, see what plugins have updates available, and then be able to individually pick and choose which plugins will get updated.

**Comment by NickRimmer** [comment]
As no one from Insomnia team reacted so far, I'm closing my PR

**Pull request closed without merging**